From dc26944591729093b22336857c446d89ae212876 Mon Sep 17 00:00:00 2001
From: Neal Gompa <ngompa13@gmail.com>
Date: Fri, 13 May 2016 10:13:44 -0400
Subject: [PATCH] Enable URPM-style solution reordering

---
 src/goal.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/goal.c b/src/goal.c
index 1a2198d..ad837bf 100644
--- a/src/goal.c
+++ b/src/goal.c
@@ -220,6 +220,8 @@ init_solver(HyGoal goal, int flags)
     solver_set_flag(solv, SOLVER_FLAG_BEST_OBEY_POLICY, 1);
     /* support package splits via obsoletes */
     solver_set_flag(solv, SOLVER_FLAG_YUM_OBSOLETES, 1);
+    /* MAGEIA: support URPM-style solution reordering */
+    solver_set_flag(solv, SOLVER_FLAG_URPM_REORDER, 1);
 
     return solv;
 }
-- 
2.6.3

